app/controllers/thread_controller.php

```diff
@@ -0,0 +1,47 @@
+<?php
+class ThreadController extends AppController
+{
+	public function index()
+	{
+		session_start();
+		
+		$page = Param::get('page', 1);
+		$per_page = 15;
```

Comment:
Please avoid using magic numbers po. You may declare it as constants :bow: 
Please apply to all